devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -297,6 +299,147 @@ public void execute() throws MojoFailureException, MojoExecutionException {
                 args.add("-Xverify:none");
             }
 
+            DevModeRunner runner = new DevModeRunner(args);
+
+            runner.prepare();
+            runner.run();
+            long nextCheck = System.currentTimeMillis() + 100;
+            Map<Path, Long> pomFiles = readPomFileTimestamps(runner);
+            for (;;) {
+                //we never suspend after the first run
+                suspend = "n";
```

Comment:
Hello @stuartwdouglas, is this meant to be passed along to the new `DevModeRunner`, so that it doesn't (start and) suspend in debug mode? If so, I don't see changing this value playing a role in it, since the `args` being passed to the new `DevModeRunner`  already have a bunch of JVM `-Xdebug` options setup in it.

Reply:
Good point, I will just move all this logic into prepare()